src/subsys/ngx_subsys_lua_socket_tcp.c.tt2

```diff
@@ -4529,8 +4520,8 @@ ngx_[% subsys %]_lua_socket_cleanup_compiled_pattern(lua_State *L)
 }
 
 
-static int
-ngx_[% subsys %]_lua_req_socket(lua_State *L)
+int
+ngx_[% subsys %]_lua_tcp_req_socket(lua_State *L)
```

Comment:
What's the point of adding the "tcp" word here? I think we make make "tcp" the default. You haven't renamed other C functions like `ngx_[% subsys %]_lua_req_socket_rev_handler` yet and also it is a bit inconsistent with the naming convention used by existing functions like `ngx_[% subsys %]_lua_socket_tcp_getreusedtimes` and `ngx_[% subsys %]_lua_socket_tcp_setkeepalive` (it should be `ngx_[% subsys %]_lua_req_socket_udp` for the udp subsystem).